itest/lnd_onchain_test.go

```diff
@@ -74,6 +77,100 @@ func testChainKitGetBlockHash(ht *lntest.HarnessTest) {
 	require.Equal(ht, expected, actual)
 }
 
+// testChainKitSendOutputsAnchorReserve checks if the SendOutputs rpc prevents
+// our wallet balance to drop below the required anchor channel reserve amount.
+func testChainKitSendOutputsAnchorReserve(ht *lntest.HarnessTest) {
+	// Start two nodes supporting anchor channels.
+	args := lntest.NodeArgsForCommitType(lnrpc.CommitmentType_ANCHORS)
+
+	// NOTE: we cannot reuse the standby node here as the test requires the
+	// node to start with no UTXOs.
+	alice := ht.NewNode("Alice", args)
+	bob := ht.Bob
+	ht.RestartNodeWithExtraArgs(bob, args)
+
+	// We'll start the test by sending Alice some coins.
+	fundingAmount := btcutil.Amount(100_000)
+	ht.FundCoins(fundingAmount, alice)
+
+	// Before opening the channel we ensure that the nodes are connected.
+	ht.EnsureConnected(alice, bob)
+
+	// We'll get the anchor reserve that is required for a single channel.
+	reserve, _ := alice.RPC.WalletKit.RequiredReserve(
```

Comment:
Can we add `RequiredReserve` to `wallet_kit.go` and access it via `alice.RPC.RequiredReserve` to avoid checking the RPC error?